.travis.yml

```diff
@@ -15,12 +11,6 @@ env:
 matrix:
   fast_finish: true
   include:
-    - php: 5.3
-      dist: precise
-    - php: 7.1
-      env: COMPOSER_FLAGS="--prefer-lowest" SYMFONY_DEPRECATION_HELPER=weak
-    - php: 5.6
-      env: DEPENDENCIES="symfony/lts:^2"
     - php: 7.1
       env: DEPENDENCIES="symfony/lts:^3"
```

Comment:
You dropped support for Symfony 3, so this line must be updated. Also `symfony/lts` is deprecated, you should set `extra.symfony.require` in `composer.json` instead (you can use `jq` to do this). See https://github.com/symfony/lts.

Reply:
While we're upgrading the Travis file, you should also add PHP 7.3 to the matrix.

Reply:
I've added php 7.3 to the matrix.

I've also COMPOSER_FLAGS="--prefer-lowest", I did need to change the phpunit-bridge version to 4.4 for this, otherwise it tried to get phpunit from https://github.com/sebastianbergmann/phpunit/archive/5.7.zip but this version isn't there anymore (https://travis-ci.org/stof/StofDoctrineExtensionsBundle/jobs/618059387#L391)

I don't understand what you mean with "use jq"